pom.xml

```diff
@@ -192,6 +192,26 @@
                     <testTarget>${java-version}</testTarget>
                 </configuration>
             </plugin>
+            <plugin>
+			    <groupId>org.apache.maven.plugins</groupId>
+			    <artifactId>maven-javadoc-plugin</artifactId>
+			    <version>3.0.0</version>
+			    <configuration>
+			        <doclet>com.iota.mdxdoclet.MDXDoclet</doclet>
+			        <sourcepath>src/main/java</sourcepath>
+			        <useStandardDocletOptions>false</useStandardDocletOptions>
+			        <destDir>docs</destDir>
+			        <additionalOptions>
+			            <additionalOption>-version "${project.version}"</additionalOption>
+			        </additionalOptions>
+			        <quiet>true</quiet>
+			        <docletArtifact>
+			            <groupId>com.iota</groupId>
+			            <artifactId>mdxdoclet</artifactId>
+			            <version>0.1</version>
```

Comment:
The examples generated are not JSON compliant:
```JSON
command = {"command": "getTrytes", "hashes": "hashes": "P9KFSJVGSPLXAEBJSHWFZLGP9GGJTIO9YITDEHATDTGAFLPLBZ9FOFWWTKMAZXZHFGQHUOXLXUALY9999"}
```
https://github.com/kwek20/iota-mdx-doclet/blob/e6655b6a4e7c147645ef2d2a0756bdadc27d11aa/examples/getTrytes.mdx#L18

Reply:
Yes it is